src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2730,15 +2872,15 @@ else if (value.isSetNullFlavor()) {
 	}
 
 	@Override
-	public Bundle tPatientRole2Patient(PatientRole cdaPatientRole) {
+	public IEntryResult tPatientRole2Patient(PatientRole cdaPatientRole, IBundleInfo bundleInfo) {
+		EntryResult result = new EntryResult();
+
 		if (cdaPatientRole == null || cdaPatientRole.isSetNullFlavor())
-			return null;
+			return result;
```

Comment:
I made a similar comment on James' PR so I want to know, what is the benefit of returning a hollow result over null?